File: plugins/OrchidChangelog/src/main/java/com/eden/orchid/languages/components/ChangelogVersionPicker.java
package com.eden.orchid.languages.components;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.theme.components.OrchidComponent;
import com.eden.orchid.languages.model.ChangelogModel;
import lombok.Getter;

import javax.inject.Inject;

public final class ChangelogVersionPicker extends OrchidComponent {

    @Getter
    private final ChangelogModel model;

    @Inject
    public ChangelogVersionPicker(OrchidContext context, ChangelogModel model) {
        super(context, "changelogVersionPicker", 100);
        this.model = model;
    }

}

File: plugins/OrchidChangelog/src/main/kotlin/com/eden/orchid/changelog/components/ChangelogVersionPicker.kt
package com.eden.orchid.changelog.components

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.options.annotations.Description
import com.eden.orchid.api.theme.components.OrchidComponent
import com.eden.orchid.changelog.model.ChangelogModel
import javax.inject.Inject

@Description("Show a list of all versions in your changelog", name = "Changelog Versions")
class ChangelogVersionPicker
@Inject
constructor(
        context: OrchidContext,
        val model: ChangelogModel
) : OrchidComponent(context, "changelogVersionPicker", 100)

